File: AlexanderKlimovLabs/2.HelloKitty/JavaVersion/HelloKitty/app/src/main/java/com/github/varenytsiamykhailo/alexanderklimov/hellokitty/MainActivity.java
package com.github.varenytsiamykhailo.alexanderklimov.hellokitty;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView mHelloTextView;

    private EditText editTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mHelloTextView = findViewById(R.id.textView);
        editTextView = findViewById(R.id.editTextTextPersonName);
    }

    public void onButtonPressed(View view) {
        if (editTextView.getText().length() == 0) {
            mHelloTextView.setText("Hello Kitty!");
        } else {
            mHelloTextView.setText("Hi, " + editTextView.getText());
        }
    }
}

File: AlexanderKlimovLabs/2.HelloKitty/KotlinVersion/HelloKitty/app/src/main/java/com/github/varenytsiamykhailo/alexanderklimov/hellokitty/MainActivity.kt
package com.github.varenytsiamykhailo.alexanderklimov.hellokitty

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.EditText
import android.widget.ImageButton
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    private lateinit var mHelloTextView: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mHelloTextView = findViewById(R.id.textView)
        val imageButton: ImageButton = findViewById(R.id.imageButton)
        val editTextView: EditText = findViewById(R.id.editTextTextPersonName)
        // альтернативный вариант
        // val imageButton = findViewById<ImageButton>(R.id.imageButton)
        // или val imageButton = findViewById(R.id.imageButton) as ImageButton

        imageButton.setOnClickListener {
            if (editTextView.text.isEmpty()) {
                mHelloTextView.text = "Hello Kitty"
            } else {
                mHelloTextView.text = "Hi, " + editTextView.text
            }
        }

    }
}